.github/workflows/PR.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: PR - Build and Test
+
+on: pull_request
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - name: Build and test
+        run: |
+          git config --global user.email "npm@dxos.org"
+          git config --global user.name "DXOS"
+
+          yarn install
+          yarn ci
```

Comment:
what does `yarn ci` do? I don't see packages.json entry for that. There is `npm ci` but I havent' heard of yarn equiv.

Reply:
ah, there is another PR open. I'll review that one first.